modules/compute/container-registry/.bicep/nested_rbac.bicep

```diff
@@ -0,0 +1,59 @@
+param description string = ''
+param principalIds array
+param principalType string = ''
+param roleDefinitionIdOrName string
+param resourceId string
+
+var builtInRoleNames = {
```

Comment:
Let's make another note that we will refactor this to move this to a security/rbac module to avoid duplication